prow/config.yaml

```diff
@@ -10319,7 +10319,7 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
-- cron: "0 0 31 2 *" # manual job, set to Feb.31st so it will never be triggered
+- interval: 20m
```

Comment:
and this becomes a gke-large-updown job essentially?

Reply:
Yeah... You could say that.